src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public interface FruitService {
+    void add(FruitTransaction transaction);
+
+    void addAll(List<FruitTransaction> transactions);
+
+    Map<String, Integer> getAll();
```

Comment:
it should contain 1 method `processData(List<FruitTransaction> transaction)` that should do the same as your addAll + add. Because you don't add data, you are processing data. It could be a return or supply